Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -5900,6 +5901,13 @@ namespace AzToolsFramework
             for (auto componentEditor : m_componentEditors)
             {
                 componentEditor->EnteredComponentMode(componentModeTypes);
+
+                // if this component editor is active and editable during component mode
+                if (componentEditor->IsSelected())
+                {
+                    // scroll to the relevant component card
+                    m_gui->m_componentList->verticalScrollBar()->setValue(componentEditor->pos().y());
```

Comment:
Just double-checking, it seemed like this might have some issues with scrolling to the right location for some combinations of components and overall Editor window size. Are those all working correctly now?

If not, maybe this should be using m_componentList->ensureWidgetVisible() or something instead...

Reply:
Agreed it feels like we should be factoring the overall visible scroll area size, the component card size, the total scrollable distance (including wrapped area that's offscreen) and the current position - hopefully you can access those values from Qt.

Reply:
I changed it to use ensureWidgetVisible and it works consistently now :)